packages/sns/src/errors/ledger.errors.ts

```diff
@@ -0,0 +1,9 @@
+import type { TransferError } from "../../candid/icrc1_ledger";
+
+export class SnsTransferError extends Error {
+  public type: TransferError;
```

Comment:
`type` isn't a reserved keyword?

Reply:
done

Reply:
Oh good to know, thanks for double checking and feedback!